views/postgres/submenu.erb

```diff
@@ -0,0 +1,30 @@
+<aside class="lg:col-span-4 xl:col-span-3 2xl:col-span-2">
+  <nav class="space-y-1">
+    <% [
+        ["Overview", "overview", "hero-squares-plus"],
+      ].each do |label, page, icon| %>
+      <% if @page == page %>
```

Comment:
Up to you, but I think this would be better if you had a separate conditionals around the attributes that may change depending on current page, as opposed to duplicating the entire `a` element.